### PR TITLE
bug(ops): flatten convict configs per ops request

### LINKED ISSUE #220: flatten config.js for ops sanity + remove env vars

### PR DESCRIPTION
**Pull request by 6a68**
Fixes #220 

Seems to be working for me locally. @pdehaan mind taking a look?

### CODE REVIEW

File: bin/create_test_data.js [hunk 1]
```diff
@@ -62,9 +62,9 @@ function createTestData(cb) {
 
   // for now, just wait 60 seconds, then fire the callback blindly
```
Comment (pdehaan) [inline bin/create_test_data.js]: update comment to "30"

File: server/work-queue/jobs/send-welcome-email.js [hunk 2]
```diff
@@ -15,11 +15,11 @@ var transporter = nodemailer.createTransport();
...
   transporter = nodemailer.createTransport(smtp({
-    host: config.get('email.smtp.host'),
-    port: config.get('email.smtp.port'),
+    host: config.get('email_smtp_host'),
+    port: config.get('email_smtp_port'),
     auth: {
-      user: config.get('email.smtp.auth_user'),
-      pass: config.get('email.smtp.auth_pass')
+      user: config.get('email_smtp_auth_user'),
+      pass: config.get('email_smtp_auth_pass')
```
Comment (pdehaan) [inline server/work-queue/jobs/send-welcome-email.js]: Need to update `config.get('email.fromEmail')` on line 34 as well.

File: bin/create_test_data.js (hunk 1 above)
Comment (6a68) [inline bin/create_test_data.js]: :-)

**Comment by 6a68** [comment]
grrr travis

File: server/work-queue/jobs/send-welcome-email.js (hunk 2 above)
Comment (6a68) [inline server/work-queue/jobs/send-welcome-email.js]: nice catch! fixed

**Comment by 6a68** [comment]
I guess travis is really unhappy with this patch. I'll look into it more deeply tomorrow.

**Comment by 6a68** [comment]
oh wait, we're red on master, too. merging, I'll look into travis failures tomorrow

**Pull request merged**